planet/cli/session.py

```diff
@@ -0,0 +1,11 @@
+"""CLI HTTP/auth sessions."""
+
+from planet.auth import AuthType
+from planet.http import Session
+
+
+class CliSession(Session):
+
+    def __init__(self, auth: AuthType = None):
+        super().__init__(auth)
+        self._client.headers.update({'X-Planet-App': 'python-cli'})
```

Comment:
We might consider making this header a class attribute @jreiberkyle ? Then we wouldn't need to override the constructor like I'm doing here. Not a big deal, though.